hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/records/TokenBurnStreamBuilder.java

```diff
@@ -23,7 +23,7 @@
  * A {@code RecordBuilder} specialization for tracking the side effects of a {@code TokenBurn}
```

Comment:
Maybe we could also improve the JavaDocs for clarity, as here it's stated as "RecordBuilder."

Reply:
Definitely...did a global replace of `{@code RecordBuilder}` with `{@code StreamBuilder}`...realized there are also a huge number of occurrences of the string "record builder"...will change those more opportunistically. 😓